src/sage/calculus/expr.py

```diff
@@ -0,0 +1,204 @@
+r"""
+Constructor for symbolic expressions
+"""
+
+from sage.misc.lazy_import import lazy_import
+
+lazy_import('sage.modules.free_module_element', ['vector', 'FreeModuleElement'])
+lazy_import('sage.matrix.constructor', 'matrix')
```

Comment:
Not a big deal (since we're not importing `*` from this file), but also `del lazy_import` here?

Reply:
No, I really only want to do that in file from which we import `*`